src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -60,13 +99,23 @@ object KubernetesGenerator {
       DeploymentSpec(
         service.replicas,
         Selector(Labels(service.name, GenType.Deployment, isDb = false)),
+        strategy = None,
         Template(
           Metadata(service.name, Labels(service.name, GenType.Deployment, isDb = false)),
           PodSpec(
             service.name,
-            Seq(Container(service.image, service.name, service.ports.map(x => ContainerPort(x._2)))),
+            Seq(
+              Container(
+                service.image,
+                service.name,
+                service.ports.map(x => ContainerPort(x._2)),
```

Comment:
Not a fan of the`._2`, doesn't really mean much. How about we have a type with some named arguments? @xsanda